content/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-3.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Fast data processing pipeline for predicting flight delays using Apache
+  APIs: Kafka, Spark Streaming and Machine Learning (part 3)"
+date: 2021-10-11T21:12:59.805Z
+author: Carol McDonald
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapR
+---
+```
+"authorDisplayName": "Carol McDonald",
+"tags": "use-cases",
+"publish": "2018-02-08T12:00:00.000Z"
+```
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/data-fabric.html>
+
+**\*Editor's Note:** This is a 3-Part Series, see the previously published posts below:*
+
+* [Part 1 - Spark Machine Learning](https://developer.hpe.com/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-/)
+* [Part 2 - Kafka and Spark Streaming](https://developer.hpe.com/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-2/)
+
+Fast Data Processing Pipeline for Predicting Flight Delays Using Apache APIs: Kafka, Spark Machine Learning, Drill, with MapR Event Store and MapR Database JSON (Part 3)
```

Comment:
Not sure if this sentence is actually needed here. You may want to delete this sentence (Fast Data Processing Pipeline....(Part 3)).

Reply:
done